### PR TITLE
 Log more config errors

### PR DESCRIPTION
**Pull request by ThatcherK**

### CODE REVIEW

File: src/core/charts/Charts.js [hunk 1]
```diff
@@ -44,6 +44,18 @@ const validConfigs = (config) => {
...
+  }
+
+  if (!(config.data || config.url || config.dataID)) {
+    window.console.error('Invalid chart config: one of data/url/dataID is required!');
+
+    return false;
+  }
+
   if (!config.mapping.subCounty) {
     window.console.warn('Potentially invalid chart config: mapping.subCounty may be required!');
```
Comment (wakibi) [inline src/core/charts/Charts.js:61]: Maybe add these in a validation function? I see it repeated and it's rather long lines of code

**Review by wakibi — CHANGES_REQUESTED** [review]

File: src/core/charts/Charts.js (hunk 1 above)
Comment (ThatcherK) [reply src/core/charts/Charts.js:61]: They are already in a function

File: src/core/utils/tables.js [hunk 2]
```diff
@@ -87,7 +87,19 @@ export const validConfigs = (config) => {
...
+  }
+
+  if (config.dataID && !config.baseAPIUrl) {
+    window.console.error('Invalid table config: baseAPIUrl is required!');
+
+    return false;
+  }
+
+  if (!(config.data || config.url || config.dataID)) {
+    window.console.error('Invalid table config: one of data/url/dataID is required!');
```
Comment (wakibi) [inline src/core/utils/tables.js:102]: I see repetitions here, so no function?

Comment (ThatcherK) [reply src/core/utils/tables.js:102]: Hmm I don't see the repeatition, the error messages are different, and the config variables checked for are also different

Comment (wakibi) [reply src/core/utils/tables.js:102]: Ohh, I see

**Review by wakibi — APPROVED** [review]

File: src/core/utils/tables.js (hunk 2 above)
Comment (ThatcherK) [reply src/core/utils/tables.js:102]: There are two validation functions, one for tables and another for charts

**Pull request merged**